package.json

```diff
@@ -126,6 +131,7 @@
     "classnames": "^2.2.6",
     "date-fns": "^1.30.1",
     "flat": "^4.1.0",
+    "gitbook-plugin-search-plus": "^1.0.3",
```

Comment:
Why adding this?

Reply:
Ug, this is getting added automatically by some process. I haven't been able to figure it out yet. Maybe @kbouchard knows.